### DISCUSSION TITLE
增加腾讯文档读取支持

### DISCUSSION THREAD

**Discussion by zmdyy0318** [mamoe/mirai#813] [Ideas]
需求应该不高，大佬们参考着弄。
某些腾讯文档只供QQ群内成员读写，通过http可以获取对应腾讯文档json形式数据，只能读不能写，但需要QQ号登录状态。
参考网址：
https://pbihub.cn/blog/1121
大致流程：
1、mirai使用cookie请求对应文档ID
2、原封不动返回数据

**Comment by sandtechnology** [comment]
问题是为什么需要读？使用网页端不好吗？
EDIT：就算只限制QQ群内成员读写，也可以保存到腾讯文档内的（

**Reply by zmdyy0318** [reply]
使用场景：
使用QQ机器人管理3个群，其中2个被管理群1个管理员群，文档记录了2个被管理群的成员列表，包括QQ号、加退群历史、论坛ID等。在管理员群里发送成员核对指令，可以通过文档内已记录成员QQ与群内现有成员进行比较，算出差异发送在管理群。
疑问：
“也可以保存到腾讯文档内的”不太理解，自己语文太差了，可以展开说一下嘛。

**Reply by sandtechnology** [reply]
啊这 diff这种事情应该记录到数据库内然后diff啊 使用现在的API就可以了

**Reply by zmdyy0318** [reply]
数据库没前端，懒，腾讯文档方便哈哈哈

**Reply by sandtechnology** [reply]
再怎么懒也可以搞个文件存json嘛 为啥要用在线服务写（
